release/release_workflow.wdl

```diff
@@ -42,133 +35,309 @@ task do_release {
+        curl --data "$API_JSON" https://api.github.com/repos/~{organization}/cromwell/releases?access_token=~{githubToken} -o release_response
 
         # parse the response to get the release id and the asset upload url
         RELEASE_ID=$(python -c "import sys, json; print json.load(sys.stdin)['id']" < release_response)
         UPLOAD_URL=$(python -c "import sys, json; print json.load(sys.stdin)['upload_url']" < release_response)
 
-        CROMWELL_UPLOAD_URL=$(sed 's/{.*}/?name=cromwell-${newVersion}.jar/' <<< "$UPLOAD_URL")
-        WOMTOOL_UPLOAD_URL=$(sed 's/{.*}/?name=womtool-${newVersion}.jar/' <<< "$UPLOAD_URL")
+        CROMWELL_UPLOAD_URL=$(sed 's/{.*}/?name=~{cromwellJarName}/' <<< "$UPLOAD_URL")
+        WOMTOOL_UPLOAD_URL=$(sed 's/{.*}/?name=~{womtoolJarName}/' <<< "$UPLOAD_URL")
 
         # Upload the cromwell jar as an asset
-        curl -X POST --data-binary @${cromwellJar} -H "Authorization: token ${githubToken}" -H "Content-Type: application/octet-stream" "$CROMWELL_UPLOAD_URL"
+        curl -X POST --data-binary @~{cromwellJar} -H "Authorization: token ~{githubToken}" -H "Content-Type: application/octet-stream" "$CROMWELL_UPLOAD_URL"
 
         # Upload the womtool jar as an asset
-        curl -X POST --data-binary @${womtoolJar} -H "Authorization: token ${githubToken}" -H "Content-Type: application/octet-stream" "$WOMTOOL_UPLOAD_URL"
+        curl -X POST --data-binary @~{womtoolJar} -H "Authorization: token ~{githubToken}" -H "Content-Type: application/octet-stream" "$WOMTOOL_UPLOAD_URL"
 
         # Publish the draft
-        curl -X PATCH -d '{"draft": false}' https://api.github.com/repos/${organization}/cromwell/releases/"$RELEASE_ID"?access_token=${githubToken}
+        curl -X PATCH -d '{"draft": false}' https://api.github.com/repos/~{organization}/cromwell/releases/"$RELEASE_ID"?access_token=~{githubToken}
     >>>
     runtime {
         docker: "python:2.7"
     }
+    output {
+        String cromwellReleaseUrl = "https://github.com/${organization}/cromwell/releases/download/${newVersion}/${cromwellJarName}"
+        String womtoolReleaseUrl = "https://github.com/${organization}/cromwell/releases/download/${newVersion}/${womtoolJarName}"
+    }
 }
 
-workflow release_cromwell {
-  String githubToken
-  String organization
+task releaseHomebrew {
+    input {
+        String organization
+        String githubToken
+        
+        String releaseVersion
+        
+        String cromwellReleaseUrl
+        String womtoolReleaseUrl
+        
+        File cromwellJar
+        File womtoolJar
+    }
+    
+    String branchName = "cromwell-${releaseVersion}"
+    
+    # This is to allow for testing. If the organization is not broadinstitute, meaning we're doing a test,
+    # instead of creating the PR in the official homebrew repo make a PR in the test repo.
+    String headBranch = if (organization == "broadinstitute") then "broadinstitute:${branchName}" else branchName
```

Comment:
I'd encourage `~{}` in `String` interpolations too

Reply:
👍 Would be awesome if the intelliJ plugin detected those as variables btw :)